### ISSUE TITLE
printable view page not showing data in latest firefox version 

### ISSUE THREAD

**Issue by AditiAvalkar-GSLab** [marcelduran/yslow#104]
printable view page not showing data in latest firefox version
(26.0)

**Comment by rohannog** [comment]
Hi,

I'm using Firefox 26 and don't see this issue. Do you have firebug installed?

Thanks.

**Comment by SebastianZ** [comment]
I can clearly reproduce this using Firefox 28.0 + Firebug 1.12.7 + YSlow 3.1.8.

Test case:
1. Open Firebug on this page
2. Switch to the _YSlow_ panel
3. Click the _Run Test_ button within the _YSlow_ panel
4. Click the _Printable View_ link within the _YSlow_ panel
5. Click _OK_ in the appearing dialog

=> A new tab will be opened, which only displays the header, the link to the page and the copyright info:

![yslowprintableview](https://cloud.githubusercontent.com/assets/958943/2621326/ea545122-bc51-11e3-9594-6b095aeb3c88.jpg)

Sebastian

**Comment by AditiAvalkar-GSLab** [comment]
Yes looks like this bug is not yet fixed and its reproducible with latest firefox version- 28.0

**Comment by filip-van-hoeckel** [comment]
Still not fixed (4 months later)

**Comment by kenficara** [comment]
Bump? This is still not working.
